source/_components/geo_location.usgs_earthquakes_feed.markdown

```diff
@@ -36,6 +36,7 @@ To integrate the U.S. Geological Survey Earthquake Hazards Program feed, add the
 geo_location:
   - platform: usgs_earthquakes_feed
     feed_type: 'past_day_all_earthquakes'
+    radius: 50
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps users to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

Reply:
OK, I'll add a full config with defaults to show expected input.